website/src/app/kb/automate/docker-compose/readme.mdx

```diff
@@ -0,0 +1,67 @@
+import Alert from "@/components/DocsAlert";
+
+# Deploy Firezone using Docker Compose
+
+Firezone Gateways can be deployed using Docker Compose. This configuration uses
+the same options shown in the Docker run command shown in the admin portal when
+deploying a Docker-based Gateway.
+
+## Prerequisites
+
+1. [Docker Engine](https://docs.docker.com/get-docker/) for Linux. Docker
+   Desktop can be used for testing purposes but is not supported for production
+   deployments.
+1. [Compose plugin for Docker Engine](https://docs.docker.com/compose/install/#scenario-two-install-the-compose-plugin)
+   version 2.0.0 or later. We recommend using the latest version available.
+
+## Deployment
+
+Use the following `docker-compose.yml`, customizing as needed:
+
+```yaml
+version: "3.7"
```

Comment:
Not a huge deal, but `version` has been deprecated
https://docs.docker.com/compose/compose-file/04-version-and-name/#version-top-level-element-obsolete

Reply:
👍